TodoCards-BackEnd/decks.py

```diff
@@ -126,4 +143,28 @@ def get_sharecode(mydb, deck_id, username):
 # then a new access record onto access table
 # returns True if success, false otherwise
 def recieve_sharecode(mydb, sharecode, username):
-    return "Unimplemented"
\ No newline at end of file
+    return "Unimplemented"
+
+
+def create_deck(mydb, deck_info, access_info, username):
+    mycursor = mydb.cursor()
+    mycursor.execute(
+        """
+        INSERT INTO `deck` (`deckName`, `deckDescription`) 
+        VALUES (%s, %s)
+        """,
+        (deck_info["deckName"], deck_info["deckDescription"])
+    )
+
+    deck_id = mycursor.lastrowid
```

Comment:
I'm curious with this access type. I would like to talk with you about this later on...